From 26e92a521fda97ef9c2eee3f810d7c74385ab97d Mon Sep 17 00:00:00 2001
From: Quanyang Wang <quanyang.wang@windriver.com>
Date: Tue, 6 Dec 2022 13:59:51 +0800
Subject: [PATCH] aptiv: set S32_VR5510 for aptiv CVC platform

The SDK commit e7f5265e1 ("plat: nxp: add dummy implementation for PMIC
callbacks") introduce S32_VR5510 to enable vr5510 driver. So we need to
enable S32_VR5510 in CVC's mk to include vr5510 driver just as what
other s32g2 platforms do.

Upstream-Status: Inappropriate [WR Linux specific change]

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 plat/nxp/s32/s32g/s32g2/aptiv.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/plat/nxp/s32/s32g/s32g2/aptiv.mk b/plat/nxp/s32/s32g/s32g2/aptiv.mk
index 9bed6e6e3..5c874594a 100644
--- a/plat/nxp/s32/s32g/s32g2/aptiv.mk
+++ b/plat/nxp/s32/s32g/s32g2/aptiv.mk
@@ -6,6 +6,8 @@
 
 S32_PLAT_SOC := s32g2
 
+S32_VR5510 := 1
+
 include plat/nxp/s32/s32g/s32g_common.mk
 
 PLAT_SOC_PATH	:= ${S32_SOC_FAMILY}/${S32_PLAT_SOC}
-- 
2.36.1

